README.md

```diff
@@ -11,99 +11,203 @@
-
-With data API builder, database objects can be exposed via REST or GraphQL endpoints so that your data can be accessed using modern techniques on any platform, any language, and any device. With an integrated and flexible policy engine, native support for common behavior like pagination, filtering, projection and sorting, the creation of CRUD backend services can be done in minutes instead of hours or days, giving developers an efficiency boost like never seen before.
+## Getting Started
 
-Data API builder is Open Source and works on any platform. It can be executed on-premises, in a container or as a Managed Service in Azure, via the new [Database Connection](https://learn.microsoft.com/azure/static-web-apps/database-overview) feature available in Azure Static Web Apps.
+To get started quickly with Data API builder, use the [Getting Started](https://learn.microsoft.com/azure/data-api-builder/get-started/get-started-with-data-api-builder) tutorial; that will help to get familiar with some basic tools and concepts while giving you a good experience on how much Data API builder can make you more efficient, but removing the need to write a lot of plumbing code.
 
-![Data API builder Architecture Overview Diagram](./docs/media/dab-architecture-overview.png)
+**1. Install the DAB CLI**
 
-## Features
+The [Data API builder (DAB) CLI](https://aka.ms/dab/docs) is a cross-platform dotnet tool. This means you will need to install the cross-platform [.NET SDK CLI](https://get.dot.net) before running the command indicated below.
 
-- Allow collections, tables, views and stored procedures to be accessed via REST and GraphQL
-- Support authentication via OAuth2/JWT
-- Support for EasyAuth when running in Azure
-- Role-based authorization using received claims
-- Item-level security via policy expressions
-- REST
-  - CRUD operations via POST, GET, PUT, PATCH, DELETE
-  - Filtering, sorting and pagination
-- GraphQL
-  - Queries and mutations
-  - Filtering, sorting and pagination
-  - Relationship navigation
-- Easy development via dedicated CLI
-- Full integration with Static Web Apps via Database Connection feature when running in Azure
-- Open Source
+```
+dotnet tool install microsoft.dataapibuilder -g
+```
 
-## Getting Started
+**2. Create your initial configuration file**
 
-To get started quickly with Data API builder for Azure Databases, you can use the [Getting Started](https://learn.microsoft.com/azure/data-api-builder/get-started/get-started-with-data-api-builder) tutorial, that will help to get familiar with some basic tools and concepts while giving you a good experience on how much Data API builder for Azure Databases can make you more efficient, but removing the need to write a lot of plumbing code.
+[Data API builder (DAB)](https://aka.ms/dab/docs) requires only a JSON configuration file to work. This can be edited by hand or with the CLI. Developers can use `dab --help` for additional syntax options.
```

Comment:
Mention the need to set environment variable `my-connection-string` to the value of the connection string as a prerequisite before running dab.

Reply:
I see that you added this as a NOTE after the start command.